internal/kafka/internal/api/public/api/openapi.yaml

```diff
@@ -1304,8 +1302,6 @@ components:
         created_at: 2021-04-07T16:24:01+05:30
     SsoProviderExample:
       value:
```

Comment:
This is not needed
```suggestion

```

Reply:
spec verification gives error. It is needed